Jenkinsfile

```diff
@@ -60,20 +49,18 @@ pipeline {
                     stage('Integration Tests') {
                         steps {
                             sh """
-                            source $WORKSPACE/etc/profile.d/uenv.sh
```

Comment:
woudln't it make sens to define the mch_environment verision as variable, so it needs to be changed only at one place, for future updates?

Reply:
good point, implemented it now.

Reply:
perfect, thanks :)